docs/gatsby-node.js

```diff
@@ -23,6 +23,7 @@ exports.onCreateWebpackConfig = ({ actions }) => {
       // Force Gatsby to look for dependencies within the local node_modules from docs.
       modules: ['node_modules'],
       alias: {
+        react: path.resolve(__dirname, '../node_modules/react'),
```

Comment:
What about upgrading the docs version of React to match? Isn't that problem. I'm not sure about resolving up a directory.

Reply:
The issue is this: https://github.com/facebook/react/issues/13991 

There's a couple resolutions in the comments, another option would be to import evergreen from `npm` instead of from `src`